### PR TITLE
Rename local.properties.example as build.properties

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/EnGt2XDX/73-rename-local-properties-example

### CODE REVIEW

File: etc/build.properties [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-# system variables which will be used at build-time
-# Any of these values can be overwritten by copying
-# this file to etc/local.properties and manually
-# editing it.
+# build.properties
+# ----------------
+# System variables which will be used at build-time.
+# Any of these values can be overwritten by creating
+# a file called etc/local.properties and editing them
+# in this local file.
```
Comment (ximenesuk) [inline etc/build.properties]: Maybe slight rewording here?

Any of these values can be overwritten by creating a new file called etc/local.properties and then adding the variables to be overwritten to this new file.

**Comment by ximenesuk** [comment]
Other than the wording above this looks fine, no problems building and good to have the change!

**Comment by sbesson** [comment]
Description update pushed.

**Comment by ximenesuk** [comment]
Given the second commit is a change to a comment this is good to merge once Travis is happy.

**Pull request merged**